tasks/k8s/helm/flux_repositories.go

```diff
@@ -0,0 +1,72 @@
+package helm
+
+import (
+	"github.com/newrelic/newrelic-diagnostics-cli/tasks"
+)
+
+// FluxRepositories - This struct defined the sample plugin which can be used as a starting point
+type FluxRepositories struct {
+	cmdExec tasks.CmdExecFunc
+}
+
+// Identifier - This returns the Category, Subcategory and Name of each task
+func (p FluxRepositories) Identifier() tasks.Identifier {
+	return tasks.IdentifierFromString("K8s/Helm/FluxRepositories")
+}
+
+// Explain - Returns the help text for each individual task
+func (p FluxRepositories) Explain() string {
+	return "Collects Flux Helm Repositories information."
+}
+
+// Dependencies - Returns the dependencies for each task.
+func (p FluxRepositories) Dependencies() []string {
+	return []string{}
+}
+
+// Execute - The core work within each task
+func (p FluxRepositories) Execute(options tasks.Options, upstream map[string]tasks.Result) tasks.Result {
+	var (
+		res []byte
+		err error
+	)
+
+	namespace := options.Options["k8sNamespace"]
+	res, err = p.runCommand(namespace)
+	if err != nil {
+		return tasks.Result{
+			Summary: "Error retrieving pods: " + err.Error(),
```

Comment:
Shouldn't this need to be `Error retrieving Flux Helm repositories:`?

Reply:
Fixed in last commit